src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int colorNum = random.nextInt(Color.values().length);
```

Comment:
Don't make your variable names shorter, rename to `colorNumber` or `colorIndex` or just `index`